packages/zoe/test/unitTests/test-zoe.js

```diff
@@ -55,12 +55,17 @@ test(`zoe.startInstance no issuerKeywordRecord, no terms`, async t => {
   const { zoe, installation } = await setupZCFTest();
   const result = await E(zoe).startInstance(installation);
   // Note that deepEqual treats all empty objects (handles) as interchangeable.
-  t.deepEqual(result, {
-    creatorFacet: {},
-    creatorInvitation: undefined,
-    instance: result.instance,
-    publicFacet: {},
-  });
+  t.deepEqual(Object.getOwnPropertyNames(result), [
+    'creatorFacet',
+    'creatorInvitation',
+    'instance',
+    'publicFacet',
+    'adminFacet',
+  ]);
+  t.deepEqual(result.creatorFacet, {});
+  t.deepEqual(result.creatorInvitation, undefined);
+  t.deepEqual(result.instance, result.instance);
+  t.deepEqual(result.publicFacet, {});
 });
```

Comment:
Can we add an assertion for what the adminFacet should have?

Reply:
done